src/layers/group.rs

```diff
@@ -101,7 +101,11 @@ pub struct GroupLayerIter<'map> {
 
 impl<'map> GroupLayerIter<'map> {
     fn new(map: &'map Map, group: &'map GroupLayerData) -> Self {
-        Self { map, group, index: 0 }
+        Self {
+            map,
+            group,
+            index: 0,
+        }
```

Comment:
What is the logic behind these reformattings? Here it's wrapping them all on their own line, but just a few lines up you removed some wrapping to form `Ok((Self { layers }, properties))`. I could be fine with either notation, but why keep changing these around?

Reply:
No reason for it, just rustfmt doing its thing.

Reply:
Can we do a single pass of rustfmt to avoid these changes, or does it actually like to change the wrapping back and forth?